pkg/controller/node/nodecontroller.go

```diff
@@ -125,9 +127,10 @@ type NodeController struct {
 	// Lock to access evictor workers
 	evictorLock sync.Mutex
 	// workers that evicts pods from unresponsive nodes.
-	zonePodEvictor         map[string]*RateLimitedTimedQueue
-	zoneTerminationEvictor map[string]*RateLimitedTimedQueue
-	podEvictionTimeout     time.Duration
+	zoneIntolerablePodEvictor map[string]*RateLimitedTimedQueue
```

Comment:
Maybe omit the word zone, just take `untoleratedTaintEvictor`, since @gmarek suggested to use a single queue for all the pods to evict.